challenge 06: Added new markdown file which consists the link of 2 gists and pull request for newly created branch

23f2003968 opened the pull request:
pull requesting the the new branch with the newly creating mark down file with the link of 2 gists

23f2003968 committed:
Intermedite Challenge 1 : Added 2 commits into 1
Adding the Aadhirajan.md file

Added the URL of the github issue to Aadhirajan.md
Challenge 5 completed

23f2003968 committed:
Adding the Aadhirajan.md file

23f2003968 committed:
Added the URL of the github issue to Aadhirajan.md
Challenge 5 completed

23f2003968 committed:
Added new markdown file which consists the link of 2 gists

23f2003968 committed:
Merge branch 'main' of https://github.com/Aadhirajan/oss101

23f2003968 committed:
this is a signed commit

23f2003968 committed:
Commiting with git signing using gpg key

NebulaTris commented:
Create a pull request with all the challenges together , this is causing merge conflicts

23f2003968 committed:
Intermediate Challenge 05 merge conflict completed

This file is created to create merge conflict

Append text to primary branch for merge conflict

appended content to merge.txt

Accidently created a new txt file so deleteing it

23f2003968 committed:
Updated python program for linters

23f2003968 committed:
Resolving merge conflict

23f2003968 committed:
resolving merge conflicts

23f2003968 committed:
resolving

23f2003968 committed:
resolving merge conflicts

23f2003968 committed:
resolving

23f2003968 committed:
resolving

23f2003968 committed:
resolving

23f2003968 committed:
linter error

23f2003968 committed:
resolving

23f2003968 committed:
linter error
